src/OneExplorer.ts

```diff
@@ -177,3 +183,35 @@ export class OneExplorer {
         vscode.window.registerTreeDataProvider('OneExplorerView', oneTreeDataProvider));
   }
 }
+
+//
+// menu handler
+//
+
+export {handleRunOnecc};
+
+/**
+ * Function called when onevscode.run-cfg is called (when user clicks 'Run' on cfg file).
+ * @param cfgUri uri of cfg file
+ */
+function handleRunOnecc(cfgUri: vscode.Uri, logger: Logger) {
```

Comment:
Did we decided to keep `vscode.Uri` in OneNode?

Reply:
@dayo09 Could you answer?

Reply:
Yeap. I left it :-D

Reply:
https://github.com/Samsung/ONE-vscode/pull/543